src/config.rs

```diff
@@ -43,8 +43,11 @@ pub struct MiniserveConfig {
     /// Show hidden files
     pub show_hidden: bool,
 
-    /// Enable random route generation
-    pub random_route: Option<String>,
+    /// Use a specific path prefix
+    pub path_prefix: Option<String>,
```

Comment:
I prefer this to be just `String`, where absent route prefix is represented as empty string. This would make it easy to use when building urls.

You may want to take a look at my previous PR https://github.com/svenstaro/miniserve/pull/534/commits/7ea3beb8de4ec134fb33b6808632d6a69d4e8161